### ISSUE TITLE
divide-by-0 error in scheduler update

### ISSUE THREAD

**Issue by rcoup** [ccnz/Ushahidi_Web#10]
Not sure why it's happening, but it crashed the scheduler...

application/libraries/Distance.php:53

```
$dist = $EARTH_RADIUS_MILES
    * (-1 * atan($dist / sqrt(1 - $dist * $dist)) + M_PI / 2);
}
```

Hack fix:

```
$dist = $EARTH_RADIUS_MILES
    * (-1 * atan($dist / (sqrt(1 - $dist * $dist) || 1)) + M_PI / 2);
}
```

**Comment by sminnee** [comment]
This also appears here: http://dev.ushahidi.com/issues/show/1572

**Comment by sminnee** [comment]
Fixed hacky solution to the Distance.php bug. Closed by 972da13e5038a8533cad6c5c6b9fce5fe0506f88.
